src/sentry/api/base.py

```diff
@@ -101,6 +104,29 @@ def handle_exception(self, request, exc):
     def create_audit_entry(self, request, transaction_id=None, **kwargs):
         return create_audit_entry(request, transaction_id, audit_logger, **kwargs)
 
+    def load_json_body(self, request):
```

Comment:
This could arguably be better written as middleware – I could see _other_ middleware finding this useful – but I'm going to keep it here for discoverability until that proves necessary.

Reply:
I like this method, but I'm not sure why we need it *now*. How have other endpoints been accessing JSON request data in the past without this?

Reply:
I added it because of @mitsuhiko's comment, frankly. I think most just `json.loads` it manually.